AdyenCard/Form/FormCardSecurityCodeItem.swift

```diff
@@ -9,19 +9,41 @@
 /// A form item into which a card's security code (CVC/CVV) is entered.
 internal final class FormCardSecurityCodeItem: FormTextItem {
 
+    internal enum DisplayMode {
+        case required
+        case optional
+        case hidden
+        
+        internal var isVisible: Bool {
+            switch self {
+            case .required, .optional: return true
+            case .hidden: return false
+            }
+        }
+        
+        internal var isOptional: Bool {
+            switch self {
+            case .optional: return true
+            case .hidden, .required: return false
```

Comment:
Isn't it considered optional if it's hidden as well?

Reply:
When re-reading it, it is actually confusing - it was meant to check for the actual enum value `optional` hence the naming. I refactored it to us a switch in place as it's only used once with more supporting context 👌